src/jokes.json

```diff
@@ -1,644 +1,683 @@
+		"a": "It's making HEADLINES!",
+		"form": "qa"
+	},
+	"130": {
+		"q": "Hey officer! How did the hackers escape?",
+		"a": "No idea. They just ransomware.",
+		"form": "qa"
+	},
+	"131": {
+		"q": "Why can’t data engineers become hat makers?",
+		"a": "They can only guarantee two thirds of a CAP!",
+		"form": "qa"
+	},
+	"132": {
+		"q": "How did the hippie learn about database transactions?",
+		"a": "By taking ACID",
+		"form": "qa"
+	},
+	"133": {
+		"q": "Why is it called the Dark Ages?",
+		"a": "There were a lot of KNIGHTS!",
+		"form": "qa"
+	},
+	"134": {
+		"q": "What did the Network Administrator say when they caught a nasty virus?",
+		"a": "It hurts when IP",
+		"form": "qa"
+	},
+	"135": {
+		"q": "Which programming language is the shortest?",
+		"a": "HTML. Because it doesn't have a neck between its <code>head</code> and <code>body</code>.",
+		"form": "qa"
+	},
+	"136": {
+		"q": "What good can come of 2989 witches casting a hex?",
+		"a": "None, it is always 0xBAD",
+		"form": "qa"
+	},
+	"137": {
+		"q": "Did you hear about the witch who was off by two when casting a hex?",
+		"a": "They failed to make the target DEAD and made them DEAF instead!",
+		"form": "qa"
+	},
+	"138": "I went to a street where the houses were numbered 8k, 16k, 32k, 64k, 128k, 256k and 512k.It was a trip down Memory Lane.",
+	"139": "Lisp programmers don't make prank calls. They make <code>FUNCALL</code>s",
+	"140": {
+		"q": "Why do Front-End Developers eat lunch alone?",
+		"a": "Because, they don't know how to join tables.",
+		"form": "qa"
+	},
+	"141": {
+		"q": "What advice do you give to a JS developer who has never played baseball?",
+		"a": "Try catch.",
+		"form": "qa"
+	},
+	"142": {
+		"q": "Why arrays Chuck Norris declares are of infinite size.",
+		"a": "Because Chuck Norris knows no bounds.",
+		"form": "qa"
+	},
+	"143": {
+		"q": "Why doesm't Chuck Norris doesn’t need garbage collection.",
+		"a": "Because he doesn’t call <code>.Dispose()</code>, he calls <code>.DropKick()</code>.",
+		"form": "qa"
+	},
+	"144": {
+		"q": "How did the programmer die in the shower?",
+		"a": "He read the shampoo bottle instructions: Lather. Rinse. Repeat.",
+		"form": "qa"
+	},
+	"145": "When I wrote this code, only God and I understood what I was doing. Now, God only knows.",
+	"146": {
+		"q": "What did the Java code say to the C code?",
+		"a": "You've got no class.",
+		"form": "qa"
+	},
+	"147": {
+		"q": "What is the most used language in programming?",
+		"a": "Profanity.",
+		"form": "qa"
+	},
+	"148": {
+		"q": "Why did the geek add body { padding-top: 1000px; } to his Facebook profile?",
+		"a": "He wanted to keep a low profile.",
+		"form": "qa"
+	},
+	"149": "8 bytes walk into a bar, the bartenders asks \"What will it be?\" One of them says, \"Make us a double.\"",
+	"150": "An SEO couple had twins. For the first time, they were happy with duplicate content.",
+	"151": "LISP - Lots of Infuriating Superfluous Parentheses",
+	"152": {
```

Comment:
Make this one a multi-line one, not a Q/A type.

Reply:
Number 152

Reply:
okay